app/domain/errors.rb

```diff
@@ -425,22 +425,57 @@ module AuthnJwt
         code: "CONJ00178E"
       )
 
-      InvalidUriConfiguration = ::Util::TrackableErrorClass.new(
-        msg: "Signing key uri authenticator configuration is invalid. It should configured as authenticator variables: " \
-              "one of the following: '{0-resource-name}','{1-resource-name}'",
+      FetchJwksKeysFailed = ::Util::TrackableErrorClass.new(
+        msg: "Failed to fetch JWKS from '{0-uri}'. Reason: '{1}'",
         code: "CONJ00179E"
       )
 
-      FetchJwksKeysFailed = ::Util::TrackableErrorClass.new(
-        msg: "Failed to fetch jwks from '{0-uri}'. Reason: '{1}'",
+      FetchJwksUriKeysNotFound = ::Util::TrackableErrorClass.new(
+        msg: "Fetched JWKS not found in response: '{0-base64-response}'",
         code: "CONJ00180E"
       )
 
-      FetchJwksUriKeysNotFound = ::Util::TrackableErrorClass.new(
-        msg: "Fetched jwks not found in response: {0}",
+
+      UnsupportedClaim = ::Util::TrackableErrorClass.new(
+        msg: "Claim '{0-claim}' does not support fetching the application identity",
         code: "CONJ00181E"
       )
 
+      MissingClaimValue = ::Util::TrackableErrorClass.new(
+        msg: "Claim '{0-claim}' value is empty or not found.",
```

Comment:
"Claim '{0-claim}' value is empty, or was not found in token."

Can you provide example filled in?

Reply:
Claim iss value is empty, or was not found in token.